cwms_radar_api/src/main/java/cwms/radar/datasource/SessionOfficePreparer.java

```diff
@@ -20,12 +20,18 @@ public SessionOfficePreparer(String office) {
 
     @Override
     public Connection prepare(Connection conn) {
```

Comment:
we do have a default office from the ApiServlet.service call. I was thinking the access manager could default to that. That would save the SessionOfficePrepared from needing any additional logic. (I feel like that call *should* be able to just assume it has a valid office.)